Extension/package.json

```diff
@@ -848,6 +848,13 @@
           "scope": "machine-overridable",
           "minimum": 0
         },
+        "C_Cpp.intelliSenseMemoryLimit": {
+          "type": "number",
+          "default": 4096,
+          "description": "%c_cpp.configuration.intelliSenseMemoryLimit.description%",
+          "scope": "machine-overridable",
+          "minimum": 2048
```

Comment:
Should we enable this to be lower, e.g. for Raspberry PI machines with low memory?

Reply:
I'll make it 1024 MB (1 GB). The smallest Raspberry Pi memory is 2 GB.

Reply:
Raspberry Pi 3 is 1 GB. Can the limit be 256 MB? The OS seems to start freezing when processes start using > 256 MB. Unfortunately, VS Code itself easily uses > 256 MB, so the user is likely to hit that issue before they hit that issue with cpptools-srv.